modules/ztp-sno-node-reboot-scenarios.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-sno-node-reboot-scenarios_{context}"]
+= SNO node reboot scenario
+
+On a {sno-caps} cluster and in {product-title} clusters generally a situation might arise due to a node reboot or kubelet restart where an associated device plugin restart has the affect of causing the application pod to fail with the `UnexpectedAdmissionError` error. This error is reported due to the fact that application pods that need devices come up before the pod that is serving the devices because there is no way to control the pod restart order.
+
+While this behavior is expected, it can lead to a pod remaining on the cluster  even when it has failed to deploy successfully and it will continue to report `UnexpectedAdmissionError`. This issue might cause undue concern particularly in a {sno-caps} deployment, where the option to drain the node is unavailable.
```

Comment:
Please make sure to check if it is okay to add an upstream link like I have in the text below before applying my suggestion below:

```suggestion
While this behavior is expected, it can lead to a pod remaining on the cluster even when it has failed to deploy successfully and will continue to report `UnexpectedAdmissionError`. This issue is however alleviated by the fact that application pods are typically wrapped up in a deployment, replicaset or a daemonset, ensuring that the subsequent pods are created and run successfully to ensure successful deployment of the application. 

There is ongoing work to ensure that such [pods are gracefully terminated](https://github.com/kubernetes/kubernetes/issues/118472) but while that materializes a simple command as below can be used to remove the failed pods:
`
kubectl delete pods --field-selector status.phase=Failed -n <POD_NAMESPACE>
`
This issue might cause undue concern particularly in a {sno-caps} deployment, where the option to drain the node is unavailable.
```